tests/litmus/memref-ops/var_buffer_cast.src.mlir

```diff
@@ -0,0 +1,13 @@
+// EXPECT: "Memory mismatch"
+
+// Although this conversion is correct, currently we don't fully support local block memory refinement.
+// So this test just checks whether return value mismatch.
+
```

Comment:
Maybe local block refinement will be supported soon!

Reply:
Local Memblock is now supported through https://github.com/aqjune/mlir-tv/pull/79.
So mlir-tv correctly verify this litmus tests